tests/integration/test.yml

```diff
@@ -1,5 +1,12 @@
 ---
 
+- name: Write the host key to known hosts
+  hosts: all
+  gather_facts: false
+  connection: local
+  tasks:
+      - shell: ssh-keyscan -t rsa {{ ansible_host }} >> ~/.ssh/known_hosts && cat ~/.ssh/known_hosts
```

Comment:
Since we sometimes want to run the tests locally, I would prefer not to add keys to my `known_hostts` file.  (Moreover, the current code would add the key everytime it runs, which is fine on an ephemeral Docker container, but not when running on my machine.)

Could we simply add `ANSIBLE_HOST_KEY_CHECKING=False` to the environment instead to disable host key checking in Ansible?

Reply:
I thought it was a bit ugly to disable it, but got your point 👍

Reply:
The `ssh-keyscan` approach essentially disables the host key check as well, since it unconditionally accepts the host key it receives, so we don't lose any security by disable the host key check.

Reply:
You are right 👍